fullscreen.bs

```diff
@@ -505,9 +518,15 @@ properties apply to this pseudo-element either.
 <p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match any
 <a>element</a> that has its <a>fullscreen flag</a> set.
 
-<p class="note no-backref">This makes it different from the {{Document/fullscreenElement}} API,
-which returns a <a>document</a>'s <a>fullscreen element</a>.
+<p class="note no-backref">This makes it different from the fullscreenElement API, which returns
+returns the topmost fullscreen element..
+
+For each <a>node tree</a>, if the result of <a>retargeting</a> its <a>shadow-including root</a>'s
+<a>fullscreen element</a> against the <a for="tree">root</a> of the <a>node tree</a> is not null and
+the result is in the same <a>tree</a>, <a>:fullscreen</a> pseudo-class must match the result.
 
+<p class="note no-backref">In other words, <a>:fullscreen</a> matches all the non-null results of
+{{DocumentOrShadowRoot/fullscreenElement}} API for each <a>node tree</a>.
```

Comment:
Updated the definition for `:fullscreen` so that the second note be unnecessary.